spec/commands/check_spec.rb

```diff
@@ -242,7 +242,7 @@
     it "returns success" do
       bundle "check --path vendor/bundle"
       expect(exitstatus).to eq(0) if exitstatus
-      expect(out).to eq("The Gemfile's dependencies are satisfied")
+      expect(out).to include("The Gemfile's dependencies are satisfied")
```

Comment:
Because I know someone might ask, I noticed this while testing and thought that the test is the one that should be fixed, not the code. Checking that output equals something is overly rigid, we should only be checking that it includes the text—i.e. the task was done.